test/external/handle-helpers

```diff
@@ -6,15 +6,15 @@ class Helpers < Sensu::Handler
     puts event_summary
   end
 
-  def api_request(*)
+  def api_request(*_args)
```

Comment:
don't think that the args should be optional, if no args are given it still returns true which I doubt is expected.